src/haz3lcore/statics/Translation.re

```diff
@@ -0,0 +1,402 @@
+let rec get_roc_exp = (t: TermBase.UExp.t, m: Statics.map): TermRoc.UExp.t =>
+  switch (t.term) {
+  | Invalid(_) => String("Invalid Not Implemented")
+  | EmptyHole => String("EmptyHole Not Implemented")
+  | MultiHole(_) => String("MultiHole Not Implemented")
+  | Tag(_) => String("Tag Not Implemented")
+  | Triv => Record([])
+  | Bool(b) => Bool(b)
+  | Int(n) => Int(n)
+  | Float(f) => Float(f)
+  | String(s) => String(s)
+  | ListLit(l) => ListLit(get_roc_exp_list(l, m))
+  | Fun(p, t) => Fun(get_roc_pat(p, m), get_roc_exp(t, m))
+  | Tuple(l) => Record(get_roc_exp_list(l, m))
+  | Var(token) => Var(get_name(m, token))
+  | Let(pat, def, body) =>
+    let indent_flag =
+      switch (def.term) {
+      | Let(_) => true
+      | _ => false
+      };
+    switch (get_typ_ann(pat, m)) {
+    | Some(p) =>
+      if (indent_flag) {
```

Comment:
can you add a comment explaining why you need to check the indent flag here? maybe with an example showing the issue